tools/loadTestFiles.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const gsWeirdBugData = fs.readFileSync('test/node/data/test_gs_weird_bug.png', 'binary');
+const bugDataObj = { data: gsWeirdBugData };
+
+const scenarios = fs
+  .readdirSync(path.join(__dirname, '../test/node/specs/bson-corpus'))
+  .filter(x => x.indexOf('json') !== -1)
+  .map(x =>
+    JSON.parse(fs.readFileSync(path.join(__dirname, '../test/node/specs/bson-corpus', x), 'utf8'))
+  );
+
+fs.writeFile('./tools/scenarios.json', JSON.stringify(scenarios, null, 2), function(err) {
```

Comment:
If there is an error with these, do you propagate an error code? It might be good to `process.exit(errCode);` if one of these writes fails.